---
std_msgs/Header header
imagine_common/PartInfo[] gap_array